nav2_bringup/launch/test_map.yaml

```diff
@@ -1,8 +1,6 @@
-map_type: "occupancy"
-image: "test_map.pgm"
+image: test_map.pgm
 resolution: 0.050000
-origin: [-15.400000, -12.200000, 0.000000]
-free_thresh: 0.196
-occupied_thresh: 0.65
+origin: [-10.000000, -10.000000, 0.000000]
```

Comment:
Why did you change the test map?

Reply:
I wasn't sure about the interaction of the map and the world. I had gazebo coming up with the other map, so I changed at the time to sync them up. Let me get your help to fix this correctly.

Reply:
We discussed and agreed to go with the turtlebot map.